quupe-RN/js/screens/LentItems/LentItemsContainer.js

```diff
@@ -38,6 +43,7 @@ class LentItemsContainer extends Component {
                     const itemsData = data.allItems;
```

Comment:
instead of using Text component to say "Loading", please use the ActivityIndicator instead.